flytekit/remote/remote.py

```diff
@@ -1782,8 +1721,6 @@ def execute_local_workflow(
         domain: str = None,
         name: str = None,
         version: str = None,
-        execution_name: typing.Optional[str] = None,
```

Comment:
We should still allow users to specify the name. If they don't specify it, FlyteAdmin should generate a random name instead of Flytekit.